src/app/units/states/edit/directives/unit-staff-editor/unit-staff-editor.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="unit-staff-editor">
+    <script type="text/ng-template" id="staffAvatar.html">
+        <a>
+          <span bind-html-unsafe="match.label | typeaheadHighlight:query"></span>
+        </a>
+    </script>
+    <mat-card>
+        <mat-card-header>
+            <mat-card-title>Modify Unit Staff</mat-card-title>  
+            <mat-card-subtitle>Add staff members to the unit, assigning them a convenor or tutor role.</mat-card-subtitle>
+        </mat-card-header>
+        <mat-card-content>
+            <table
+                mat-table [dataSource]="dataSource"
+                matSort
+                class="f-table"
+            >
+                <!-- Name Column-->
+                <ng-container matColumnDef="name">
+                    <th mat-header-cell *matHeaderCellDef mat-sort-header>Name</th>
+                    <td mat-cell *matCellDef="let staff">
+                      <span class="staff-name-container">
+                        <user-icon [size]="30" [user]="staff.user"></user-icon>
+                        <p>{{staff.user.name}}</p>
+                      </span>
+                    </td>
+                </ng-container>
+                
+                <!-- Role Column -->
+        <ng-container matColumnDef="role" sticky>
+            <th mat-header-cell *matHeaderCellDef mat-sort-header>Role</th>
+            <td mat-cell *matCellDef="let staff">
+              
+              <mat-button-toggle-group name="fontStyle" aria-label="Font Style" [(ngModel)]="staff.role" value="{{staff.role}}">
+                <mat-button-toggle value="Tutor"  (click)="changeRole(staff,2)" matTooltip="Tutors deliver unit content">Tutor</mat-button-toggle>
```

Comment:
Consider adding labels to the role toggle buttons for better accessibility and understanding. Users might not immediately understand the purpose of each toggle without clear labels.